pages/mensa.vue

```diff
@@ -14,25 +15,26 @@
       row>
       <v-flex
         v-for="dayPlan in weekPlan"
-        :key="dayPlan.id"
+        :key="dayPlan.date"
         xs12>
         <v-card 
           height="100%">
           <v-card-title> 
-            <h4>{{ getDayHeader(dayPlan) }}</h4>
+            <h3>{{ getDayHeader(dayPlan) }}</h3>
           </v-card-title>
-          <v-divider />
+          <v-divider/>
           <v-list 
             v-for="item in dayPlan.data"
             :key="item.id"
             dense
             two-line>
-            <v-list-tile>
-              <v-list-tile-content class="content">
-                <span class="category">{{ item.category }}:</span>
-                <span>{{ item.name }} ({{ getPriceLabel(item.prices.students) }})</span>
-              </v-list-tile-content>
-            </v-list-tile>
+            <div class="list-tile">
+              <span class="category">{{ item.category }}:</span>
+              <br>
+              <span>{{ item.name }}</span>
+              <br>
+              <span class="price">Studenten: {{ getPriceLabel(item.prices.students) }} - Angestellte: {{ getPriceLabel(item.prices.employees) }}</span>
```

Comment:
Das price label könnte man in die store-Objekte packen, damit sie nicht bei jedem Rendering neu berechnet werden müssen.